[29기 박무선] 이력서 업로드 페이지 레이아웃 작업 & 백엔드 통신

mooseonpark opened the pull request:
## :: 최근 작업 주제 (하나 이상의 주제를 선택해주세요.)
- [x] 기능 추가
- [ ] 리뷰 반영
- [ ] 리팩토링
- [ ] 버그 수정
- [ ] 컨벤션 수정

<br />

## :: 구현 목표 (해당 브랜치에서 구현하고자 하는 하나의 목표를 설정합니다.)
- 이력서 파일 업로드 및 삭제기능 구현

<br />

## :: 구현 사항 설명 (작업한 내용을 상세하게 기록합니다.)
1. 새 이력서 파일 생성 및 이력서 파일 업로드
- 새 이력서 파일 누르면 새로운 페이지에서 정보 입력 후 파일저장 가능
- 기존 로컬에 있는 pdf파일 업로드 기능 
- 리스트에 있는 이력서 파일 삭제 기능

<br />

## :: 성장 포인트 (해당 기능을 구현하며 고민했던 사항이나 새로 알게된 부분, 어려웠던 점 등을 작성합니다.)
- 백엔드와 post로 데이터 통신 송수신에 익숙해집니다.
- 가장 기본적인 CRUD 기능 구현을 연습합니다.

<br />

## :: 기타 질문 및 특이 사항
- 함수 로직이 너무 긴 것 같습니다. 좀 더 효율적인 방법이 없을지 궁금합니다.(예시)

yeonuk-hwang commented:
무선님! 이 브랜치에 기존에 작업하시던 브랜치 내용이 다 들어가있네요, 이렇게 되면 리뷰를 진행하기 어렵습니다!
기존 브랜치 머지되고 나면 리베이스 하시면 그 때 다시 한번 보겠습니다!

jujusnake committed:
Fix : index.html에 컨플릭트가 해결 안된채로 master에 머지됨 -> 해결
Fix: conflict resolved
Modify: 리뷰 반영
Modify: main 수정중
Modify: main 수정
Add: Resume page
Add: Resume file layout
Add: Resume 페이지 레이아웃 작업
Fix: error 수정
Fix : env 파일 삭제
Modify: 레이아웃 수정
change branch
Modify: 통신중
Modify: 수정
Add: 업로드 기능 구현
Fix: sider issue resolved
